packages/appium/docs/en/ecosystem/tools.md

```diff
@@ -0,0 +1,32 @@
+---
+hide:
+  - toc
+
+title: Appium-Related Tools
+---
+
+There are several Appium tools that have been created to to assist with things not directly related
+to testing, such as Appium installation, test development, and more.
+
+### Appium Inspector
+
+Appium has a graphical client which can be used to inspect application screenshots, view the
+application hierarchy, run Appium commands, record app interactions, and more. It is very useful
+for Appium test development.
+
+Learn more on its GitHub page: [Appium Inspector](https://github.com/appium/appium-inspector)
+
+### Appium Doctor
+
+The `appium-doctor` CLI tool can be used to validate proper environment setup for Android and iOS
+automation, and identify any common configuration issues.
+
+Learn more on its GitHub page: [Appium Doctor](https://github.com/appium/appium/tree/master/packages/doctor)
```

Comment:
Can we add the install command as well?
The GitHub page has the command, but we have old one, that is available `npm install -g appium-dcotor`. So I think it would be helpful to explicitly show the new one is `npm install -g @appium/doctor`

Reply:
Added

Reply:
thank you